drms/json.py

```diff
@@ -63,7 +63,11 @@ class HttpJsonRequest:
 
     def __init__(self, url, encoding):
         self._encoding = encoding
-        self._http = urlopen(url)
+        try:
+            self._http = urlopen(url)
+        except HTTPError as e:
+            msg = f"Failed to open URL: {e.url} with {e.code} - {e.msg}"
+            raise IOError(msg)
```

Comment:
This isn't the correct one but raising HTTPError with a nice message was more painful